stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/extensions/Message.kt

```diff
@@ -26,6 +27,12 @@ import java.util.Date
 @InternalStreamChatApi
 internal fun Message.isDeleted(): Boolean = deletedAt != null
 
+/**
+ * @return if the message was sent by current user.
+ */
+@InternalStreamChatApi
+public fun Message.isMine(): Boolean = ChatClient.instance().getCurrentUser()?.id == user.id
```

Comment:
I think this exists somewhere already - at the very least in each `MessageItemState`, we should be able to use it from there

Reply:
Yes, but in this case the extension function is used inside `QuotedMessage` which takes in `Message` propagated through a few of elements and refactoring it to take `MessageItemState` will introduce breaking changes which i don't think are worth replacing one extension function.